code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -1013,21 +1013,20 @@
 /datum/reagent/toxin/acid/on_hydroponics_apply(obj/item/seeds/myseed, datum/reagents/chems, obj/machinery/hydroponics/mytray, mob/user)
 	. = ..()
 	if(chems.has_reagent(type, 1))
-		mytray.adjust_plant_health(-round(chems.get_reagent_amount(type) * 1))
-		mytray.adjust_toxic(round(chems.get_reagent_amount(type) * 1.5))
-		mytray.adjust_weedlevel(-rand(1,2))
+		var/acid_dmg = acidpwr * 0.1 // results in 1 for sulphric acid, 4.2 for flurosuluric acid, and 0.5 for nitric acid
+		mytray.adjust_plant_health(-round(chems.get_reagent_amount(type) * acid_dmg))
+		mytray.adjust_toxic(round(chems.get_reagent_amount(type) * acid_dmg))
+		mytray.adjust_weedlevel(-rand(1, 1 + round(acid_dmg)))
+		mytray.adjust_pestlevel(-rand(1, 1 + round(acid_dmg))) // RIP spiderlings
 
 /datum/reagent/toxin/acid/expose_mob(mob/living/carbon/exposed_carbon, methods=TOUCH, reac_volume)
 	. = ..()
 	if(!istype(exposed_carbon))
 		return
-	reac_volume = round(reac_volume,0.1)
-	if(methods & INGEST)
-		exposed_carbon.adjustBruteLoss(min(6*toxpwr, reac_volume * toxpwr))
-		return
-	if(methods & INJECT)
-		exposed_carbon.adjustBruteLoss(1.5 * min(6*toxpwr, reac_volume * toxpwr))
+	if(methods & INGEST|INJECT)
```

Comment:
this will always be true, for example `(4 & 1 | 2) == 2`,
this should be `methods & (INGEST | INJECT)`